docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -73,11 +70,36 @@ To see the supported values for each option, run:
 
    admin/run-acceptance-tests --help
 
+Configuration File
+------------------
+
+The configuration file given for the ``--config-file`` parameter contains information about compute-resource providers and dataset configurations.
+The top-level object in the file is a mapping.
+
+It may optionally contain a ``metadata`` key.
+If it does and if the provider supports it,
+the value should be a mapping and the contents will be added as metadata of the created nodes.
+
+The top-level mapping must contain a ``dataset-backends`` item.
```

Comment:
I'm not sure that separating this from the provide configuration is useful.

For ZFS and loopback, presumably testing the defaults is enough.